streams/src/main/java/org/apache/kafka/streams/state/internals/SessionKeySchema.java

```diff
@@ -72,6 +72,27 @@ public boolean hasNext(final KeyValueIterator<Bytes, ?> iterator) {
     }
 
     @Override
+    public HasNextCondition hasNextCondition(final Bytes binaryKeyFrom, final Bytes binaryKeyTo, final long from, final long to) {
```

Comment:
As noted above - can we remove the other overload of this method and just use this one?